pre-cache/common

```diff
@@ -4,6 +4,10 @@ container_tool="${container_tool:-podman}"
 pull_secret_path="${pull_secret_path:-/host/var/lib/kubelet/config.json}"
 pull_spec_file="${pull_spec_file:-/tmp/images.txt}"
 config_volume_path="${config_volume_path:-/etc/config}"
+podman_pull_path="${podman_pull_path:-/run/containers/0}"
```

Comment:
i see this is a refactor so this should be working, but I'm curious what does /run/containers/0 contain? Is this a file that always exist on coreos in the underlying FS of an openshift cluster?

Reply:
when you do a podman login, the auth is stored there, so I assume it's just creating the path.

Reply:
No, this is a folder where podman that runs as root looks for the pull secret by default. We copy the pull secret there from the host